src/GeositeFramework/js/widgets/map_utils/measure/AgsMeasure.js

```diff
@@ -210,14 +211,32 @@ define([
                     dojo.connect(_pointLayer, 'onDblClick', handleMarkerDblClick);
             },
 
+            getLength = function(line) {
+                return _.map(options.esriLengthUnits, function (u) {
```

Comment:
I presume `u` is representing a unit variable, it would be easier to read and comprehend with a full variable name like `unit`.